pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -199,6 +199,11 @@ func runStandaloneTask(
 		return false
 	}
 
+	if !*ecsInput.RunStandaloneTask {
+		in.LogPersister.Infof("Skipped running task")
+		return true
+	}
```

Comment:
Just to confirm, with this change, the task would not be "even" created at the time piped "sync" the ECS app. We can expect other triggers to start the tasks for us, is that right? In such case, how can that trigger specify the exact taskDefinition revision to use to create the ECS task 🤔 WDYT @kentakozuka @nnnkkk7

Reply:
> We can expect other triggers to start the tasks for us, is that right? 

Yes. We need other triggers to start the task.

> In such case, how can that trigger specify the exact taskDefinition revision to use to create the ECS task

AFAIK, EventBridge Scheduler allows us to select the latest revision or specific one to run.

![Untitled (1)](https://github.com/pipe-cd/pipecd/assets/16733673/1ed993fe-3313-4392-aac5-35642e8ffe9d)

Reply:
yep, I understand that they allow us to do that, but the point is how actually dev can understand that which revision of taskdef is what they want (if they create it not of the time they commit the code change). Anw, it's just my concern on how actually this function could be used in usecase, look like it not really pipecd problem (if both of you agree with that way to select revision). Besides, maybe print out registered taskdef revision on stage log could help.